scipy/stats/_continuous_distns.py

```diff
@@ -1516,6 +1516,10 @@ def _sf(self, x, a):
         fac = 0.5*sc.gammainc(a, abs(x))
         return np.where(x > 0, 0.5-fac, 0.5+fac)
 
+    def _entropy(self, a):
+        h = a + np.log(2) + sc.loggamma(a) + (1 - a) * sc.digamma(a)
```

Comment:
```suggestion
        h = a + np.log(2) + sc.gammaln(a) + (1 - a) * sc.digamma(a)
```

Please prefer `gammaln` over `loggamma`. `loggamma` is not used for any other real distribution and slower than `gammaln`:

```
%timeit -n 100000 sc.loggamma(1) # 697 ns ± 34.9 ns per loop
%timeit -n 100000 sc.gammaln(1) # 670 ns ± 13.5 ns per loop
```